Gemfile

```diff
@@ -16,6 +18,7 @@ gem 'webpacker', '~> 5.0'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'devise'
```

Comment:
For now it looks like that comment `Build JSON APIs with ease...` belongs to gem devise. Move gem below in the code. Also, it is a good practice to add small comment with gem function - you can take it from gem documentation